src/main/java/edu/kit/ipd/crowdcontrol/objectservice/notification/NotificationPolicy.java

```diff
@@ -0,0 +1,34 @@
+package edu.kit.ipd.crowdcontrol.objectservice.notification;
+
+/**
+ * NotificationPolicy encapsulates strategies to check and send notifications.
+ *
+ * @author Simon Korz
+ * @version 1.0
+ */
+public abstract class NotificationPolicy<T> {
```

Comment:
i mean that this class could also be an interface without any change, the invoke could be a default method and you would gain a bit a bit flexibility without much effort